### ISSUE TITLE
Hundreds of "Added non-passive event listener to a scroll-blocking <some> event." warnings in Query Builder

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#14463]
Performance slows down to a crawl with the console open.

Steps:

1.  New query against Sample Dataset
2. Summarize by Count
3. Group by Created At > Month
4. Open the "Summarize" sidebar
5. Over 400 warnings in the console. Opening the sidebar took almost 10 seconds, as if it was in slow motion

> [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

![image](https://user-images.githubusercontent.com/1455846/105251385-39ba4a80-5b30-11eb-864a-74d0b11bc0ed.png)

**Comment by ariya** [comment]
Does this apply to any table in the Sample Dataset or only a particular one?

**Comment by camsaul** [comment]
Not sure... I only checked with `Orders`

**Comment by gusaiani** [comment]
Looks like this is a longstanding issue in React core, probably interacting with D3.

See:
https://github.com/facebook/react/issues/6436

Considering:

* deflagging `Verbose` from the console options hides the messages
* the console message itself does not point to parts of our code that may be causing the warnings.
* future updates of React and other libs may introduce the fix

I propose we close this issue as `.Wont Fix` for now.

@flamber @camsaul @ariya,
ok by you?

**Comment by ariya** [comment]
Yeah, I agree. Looks like it's also not as severe as the situation back then. At least with the latest master (commit `d65533abe`), I only witness a handful of such messages, probably at max just 4 and not hundreds of them anymore.

**Comment by gusaiani** [comment]
Per @flamber (via Slack), we are not closing issues with `Won’t fix` at the moment.
